js/repeater-list.js

```diff
@@ -201,7 +300,8 @@
 			list_selectable: false,
 			list_sortClearing: false,
 			list_rowRendered: null,
-			list_frozenColumns: 0
+			list_frozenColumns: 0,
+			list_itemActions: false
```

Comment:
I think we should call this option `list_actions` to simplify (replacing the name throughout the control). Then, I'd suggest making this accept an object vs an array. The object would have an `items` attribute, which would be the array of action items :) The reason for the object is I think we need other attributes here, such as `label` and `width`. Since the actions column, while fixed, is essentially an additional column, we need a means to set it's column label and width just like we have via the columns settings.